doc/project/checklist.md

```diff
@@ -345,6 +345,7 @@ A functional coverage shell object has been created - this may not contain cover
 ### PRE_VERIFIED_SUB_MODULES_V1
 
 Sub-modules that are pre-verified with their own testbenches have already reached V1 or a higher stage.
+For sub-modules that are pre-verified, but are not tracked (i.e., not taken through the verification milestones), ensure that their coverage level meets or exceeds the V2 milestone requirement.
```

Comment:
Should this be "V1 milestone" ?

Reply:
Good catch. I should add this statement to all 3 milestones. Fixed, thanks